packages/graphback-codegen-resolvers/src/templates/LayeredResolverTemplates.ts

```diff
@@ -0,0 +1,111 @@
+// TODO subscription support
+// TODO support for the database type instead of hardcoded lowercase
+export const createTemplate = (fieldName: string, typeName: string, subscription?: boolean): string => {
+  return `${fieldName}: (_, args, context) => {
+      validateContext(context)
+      return context.crudService.create("${typeName.toLowerCase()}", args.input, context);
+    }`
+}
+
+export const updateTemplate = (fieldName: string, typeName: string, subscription?: boolean): string => {
+  return `${fieldName}: (_, args, context) => {
+      validateContext(context)
+      return context.crudService.update("${typeName.toLowerCase()}", args.id, args.input, context);
```

Comment:
Because `context` is passed as `any` `context.crudService` is magically bound. Could we pass `context` as `GraphbackRuntimeContext`?

Reply:
Separate PR.